src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -2183,14 +2191,16 @@ def managed_instance_list(
 
 
 def managed_instance_update(
+        cmd,
         instance,
         administrator_login_password=None,
         license_type=None,
         vcores=None,
         storage_size_in_gb=None,
         assign_identity=False,
         proxy_override=None,
-        public_data_endpoint_enabled=None):
+        public_data_endpoint_enabled=None,
+        sku=None):
```

Comment:
In the interest of alignment & reducing overall complexity, would it make sense to use same syntax as db_update function (i.e. split it up into components):
```
        tier=None,
        family=None,
        capacity=None,
```
?

Reply:
Just remembered that MI has vcores property, not capacity. Blergh

Reply:
I'll replace sku with tier and family. I'll omit the third component of sku because it is not needed